client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -270,35 +270,29 @@ class CommitManager(appId: String, val conf: CelebornConf, lifecycleManager: Lif
     } else {
       commitHandlers.computeIfAbsent(
         partitionType,
-        (partitionType: PartitionType) => {
-          partitionType match {
-            case PartitionType.REDUCE => new ReducePartitionCommitHandler(
-                appId,
-                conf,
-                lifecycleManager.shuffleAllocatedWorkers,
-                committedPartitionInfo,
-                lifecycleManager.workerStatusTracker)
-            case PartitionType.MAP => new MapPartitionCommitHandler(
-                appId,
-                conf,
-                lifecycleManager.shuffleAllocatedWorkers,
-                committedPartitionInfo,
-                lifecycleManager.workerStatusTracker)
-            case _ => throw new UnsupportedOperationException(
-                s"Unexpected ShufflePartitionType for CommitManager: $partitionType")
-          }
+        {
```

Comment:
it's recommended by IDEA, not sure if it works on scala 2.11, let's wait for CI report